Mathlib/Topology/MetricSpace/Thickening.lean

```diff
@@ -80,6 +80,7 @@ theorem thickening_of_nonpos (hδ : δ ≤ 0) (s : Set α) : thickening δ s = 
 
 /-- The (open) thickening `Metric.thickening δ E` of a fixed subset `E` is an increasing function of
 the thickening radius `δ`. -/
+@[gcongr]
 theorem thickening_mono {δ₁ δ₂ : ℝ} (hle : δ₁ ≤ δ₂) (E : Set α) :
```

Comment:
This design decision may merit further discussion, as `thickening` is monotone in both delta and the E. I'm not sure which lemma is more useful...